tests/nightly/test_large_array.py

```diff
@@ -134,6 +134,13 @@ def test_Dense(ctx=mx.cpu(0)):
     res.wait_to_read()
     assert res.shape == (50000000, 100)
 
+def test_where():
+    a = nd.ones(shape=(LARGE_X, SMALL_Y))
+    b = nd.arange(0, LARGE_X).reshape(LARGE_X, 1)
+    b = nd.broadcast_to(b, shape=(b.shape[0], SMALL_Y))
+    res = nd.where(b > 100, a, b)
```

Comment:
Can we add test for csr data to test where_csr?

Reply:
Thanks. test added

Reply:
can we add a test for backward too.